meteor/lib/collections/PeripheralDevices.ts

```diff
@@ -64,7 +64,17 @@ export enum PlayoutDeviceType { // to match DeviceType in TSR
 	PANASONIC_PTZ = 5,
 	// TCPSEND = 6, // to be implemented
 	HYPERDECK = 7,
-	PHAROS = 8
+	PHAROS = 8,
+	SCANNER = 9,
+	LAUNCHER = 10
+}
+export const DefaultPlayoutDevicePort = {
```

Comment:
@nytamin I don't like this pattern. Unsure what will be the cleanest way.

Reply:
Yeah, I believe we should add the scanner and launcher as peripheral-devices. That gives us the flexibility of running Sofie with or without launcher & media-scanner